e2e/windows/e2e-scenario.sh

```diff
@@ -167,6 +177,7 @@ set +e
 az deployment group create --resource-group $MC_RESOURCE_GROUP_NAME \
          --template-file $DEPLOYMENT_VMSS_NAME-deployment.json || retval=$?
 set -e
+log "Group deployment creation completed"
```

Comment:
Thanks for adding the log. But "Group deployment creation" is confused. Could you help to provide a better log, for example, 'Deploying Windows VMSS' by reference line 183?